test/ably/resthttp_test.py

```diff
@@ -135,3 +137,12 @@ def test_custom_http_timeouts(self):
         self.assertEqual(ably.http.http_open_timeout, 8)
         self.assertEqual(ably.http.http_max_retry_count, 6)
         self.assertEqual(ably.http.http_max_retry_duration, 20)
+
+    def test_version_headers(self):
+        req = Request('GET', '/', skip_auth=True)
+
+        self.assertIn('X-Ably-Version', req.headers.keys())
+        self.assertIn('X-Ably-Lib', req.headers.keys())
+
+        self.assertEqual(req.headers.get('X-Ably-Version'), __api_version__)
```

Comment:
I know this sounds pedantic, but if someone mistakenly changes setup.py and does not realise that it in fact changes the headers, then the point of the test is to make them aware of this.  So I would prefer that you hard code in the expectation for protocol version, and for `X-Ably-Lib` perhaps use a regex like `0\.9\.\d+`